tests/ci/runpytest.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
+import os
+
+from azureml.core import Run
+print('before run.get_context')
+run = Run.get_context()
+print('before subprocess.run')
+
+# subprocess.run(["python", "-m", "pytest","tests/unit",
+#                "-m","not notebooks and not spark and not gpu",
+#                "--junitxml=reports/test-unit.xml"])
+# subprocess.run(["pytest","tests/unit/test_python_utils.py",
+#                "-m","not notebooks and not spark and gpu",
+#                "--junitxml=reports/test-unit.xml"])
+# subprocess.run(["pytest","tests/unit/test_dataset.py",
+#                "-m","not notebooks and not spark and not gpu",
+#                "--junitxml=reports/test-unit.xml"])
+
+subprocess.run(["pytest", "tests/unit/test_timer.py",
```

Comment:
might be more generalizable to use argparse and pass in arguments for the markers / keywords to use with the test

Reply:
agree. But, I was asked to push this out now. So, let's get this out and then I can get additional items on the todo list.

Reply:
After talking with @bethz, we agreed that we don't want to have code in the repo that doesn't follow the contribution standards. It is important to make sure that everything that we put in the repo works correctly.

I'm going to talk with the LT to make sure they understand the hard work that @bethz is putting into this.

Reply:
added argparse for parameters.